src/foam/u2/borders/MarginBorder.js

```diff
@@ -0,0 +1,41 @@
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'MarginBorder',
```

Comment:
A little uncertain about using this. 

The intercept popUp needs a UI upgrade - at the very least we will need some ux that will identify what is happening with 'pending' status capabilities. Which makes me think this is excessive for a temp solution.

Then I also wonder - considering our most recent dev-channel conversations over margins. Its just another reason I feel this is excessive.

Also - I don't see the reason to not add this as css - why a model.

Reply:
^ not a blocking comment

Reply:
> considering our most recent dev-channel conversations over margins
This is the reason I added this. `MarginBorder` allows me to take the concern of margin outside of the intercept view.

> why a model
CSS axiom would work too, but I figure if there's a CardBorder it makes sense to have a MarginBorder seeing as CardBorder also could be a CSS axiom.

Reply:
Also just remembered, if this is a CSS axiom then the margin has to be defined statically. Consider GMail's different options for the user to change padding - if we ever did something similar it would be trivial to do with MarginBorder but rather complicated to do with a MarginCSSAxiom

Reply:
I'm cool with this approach.